src/graph.jl

```diff
@@ -138,6 +148,39 @@ function status(sf::StackFrame)
     return st
 end
 
+function Base.:occursin(needle::Union{AbstractString,Regex,AbstractChar}, g::Node)
```

Comment:
Particularly the second of these seems sufficiently "strange" that perhaps it's best not to overload the Base function. How about making this an internal method? Maybe `nodematches`?

Reply:
Yes, that's reasonable.